src/resources/base.js

```diff
@@ -0,0 +1,4 @@
+/** @provideGoog */
```

Comment:
Thank Tyler! After some poking around (basically taking a real base.js and bisecting) turns out the special part that makes `goog.require` persist as a provided symbol is `var COMPILED = false;`.

@mprobst PTAL, before I update the goldens?